src/rvoice/fluid_rvoice_mixer.c

```diff
@@ -735,7 +744,10 @@ new_fluid_rvoice_mixer(int buf_count, int fx_buf_count, int fx_units, fluid_real
     
     for(i = 0; i < fx_units; i++)
     {
-        mixer->fx[i].reverb = new_fluid_revmodel(sample_rate);
+        /* cannot access to the maximum sample rate value indicated by the
+           settings. So the the value is hard coded here: 96000Hz
+        */
+        mixer->fx[i].reverb = new_fluid_revmodel(96000.0, sample_rate);
```

Comment:
In  `new_fluid_synth()`, pls use 

`fluid_settings_getnum_range(settings, "synth.sample-rate", &min, &max);`

 to read the maximum rate and then pass it on to `rvoice_mixer`.